src/app/tests/suites/OperationalCredentialsCluster.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2021 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Operational Credentials Cluster Tests
+
+config:
+    cluster: "Operational Credentials"
+    endpoint: 0
+
+tests:
+    - label: "Read number of supported fabrics"
+      command: "readAttribute"
+      attribute: "SupportedFabrics"
+      response:
+          value: 16
```

Comment:
This is not a scalable test. If it is run against a device that supports < 16 fabrics, it will fail. We need a more scalable way to test this kind of "per-product constant" attribute.

Reply:
@tcarmelveilleux I updated the ZAP tool to generate tests that can check for value limits. The new tests are updated as well.